<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameSettings</class>
 <widget class="QDialog" name="GameSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tetris</family>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>18</y>
     <width>181</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tetris</family>
    </font>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="glassHeightLabel">
      <property name="font">
       <font>
        <family>Tetris</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>界面高度</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="glassWidthLabel">
      <property name="font">
       <font>
        <family>Tetris</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>界面宽度</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="glassHeight">
      <property name="font">
       <font>
        <family>Tetris</family>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="glassWidth">
      <property name="font">
       <font>
        <family>Tetris</family>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="cellSizeLabel">
      <property name="font">
       <font>
        <family>Tetris</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>方块大小</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="cellBorderLabel">
      <property name="font">
       <font>
        <family>Tetris</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>方块间隔</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="cellBorder">
      <property name="font">
       <font>
        <family>Tetris</family>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="cellSize">
      <property name="font">
       <font>
        <family>Tetris</family>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GameSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GameSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
